### PR TITLE
add .net interactive guidances

### LINKED ISSUE #438: Not clear how to select Try.NET scaffolding
The [section about interactive code snippets](https://docs.microsoft.com/en-us/contribute/code-in-docs#code-snippets-included-by-reference) contains the following options of `interactive`:

> - `try-dotnet` - Enables Try .NET
> - `try-dotnet-class` - Enables Try .NET with class scaffolding
> - `try-dotnet-method` - Enables Try .NET with method scaffolding

However, it's not clear for which scenarios what type of scaffolding to use?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d876ecc9-4bd3-7939-e085-45ee004d2ca4
* Version Independent ID: 01ce33da-da13-b0f5-a153-3e1d41d848f5
* Content: [How to include code in docs - Contributor Guide](https://docs.microsoft.com/en-us/contribute/code-in-docs#code-snippets-included-by-reference)
* Content Source: [Contribute/code-in-docs.md](https://github.com/MicrosoftDocs/Contribute/blob/master/Contribute/code-in-docs.md)
* Product: **non-product-specific**
* GitHub Login: @tdykstra
* Microsoft Alias: **tdykstra**

### PR DESCRIPTION
**Pull request by BillWagner**
Fixes #438

The differences between the different `try-dotnet` settings was not described. Add a description and guidance for when to pick each one.

While making these edits, update to the new product name *.NET Interactive*.

### CODE REVIEW

File: Contribute/code-in-docs.md [hunk 1]
```diff
@@ -400,13 +400,19 @@ available attribute values are:
...
+* `try-dotnet-method` - Enables .NET Interactive with method scaffolding
 
 For the Azure Cloud Shell and PowerShell Cloud Shell, users can only run commands against their own
 Azure account.
 
+For the .NET Interactive experience, the contents of your code block depends on which of the three scaffolding experiences you choose:
+
+* *No scaffolding* `try-dotnet`: The code block should represent a full program text. For example, the *Program.cs* file generated by `dotnet new console` would be valid. These are most useful to show an entire small program, including the `using` directives needed. Top level statements are not supported at this time.
+* *Method scaffolding* `try-dotnet-method`: The code block should represent the content of a `Main` method in a console application. You can assume the `using` directives added by the `dotnet new console` template. This setting is most useful for short snippets that demonstrate one feature.
+* *Class scaffolding* `try-dotnet-class`: The code block should represent a class with a `Main` method as the program entry point. These can be used to show how members of a clas interact.
```
Comment (tdykstra) [inline Contribute/code-in-docs.md:414]:
```suggestion
* *Class scaffolding* `try-dotnet-class`: The code block should represent a class with a `Main` method as the program entry point. These can be used to show how members of a class interact.
```

**Review by tdykstra — APPROVED** [review]
Very helpful additions!  One typo to fix.

File: Contribute/code-in-docs.md (hunk 1 above)
Comment (BillWagner) [reply Contribute/code-in-docs.md]: I saw it the same time you did. 👍

**Comment by BillWagner** [comment]
#sign-off

**Pull request merged**